editions/tw5.com/tiddlers/concepts/Customizing Field Rendering.tid

```diff
@@ -0,0 +1,31 @@
+created: 20220110205600000
+modified: 20220110205600000
+tags: Concepts
+title: Customizing field rendering
+type: text/vnd.tiddlywiki
+
+Tiddler fields are normally rendered as a simple input boxes. To modify this behaviour, a tiddler with the name of the field can be created below `$:/core/config/EditTemplateFields/Templates`. The content of this tiddler is transcluded to represent the content of this field.
```

Comment:
Should this be `$:/core/config/EditTemplateFields/Templates` or `$:/config/EditTemplateFields/Templates` ?